.travis.yml

```diff
@@ -216,11 +216,12 @@ matrix:
           - /root/travis_bootstrap
           - /hab/studios/home--travis--build--habitat-sh
           - /root/.cargo
-      before_install:
+      script:
         - ./support/ci/fast_pass.sh || exit 0
+      before_deploy:
         - if [[ ! -x ./support/ci/deploy.sh ]]; then chmod +x ./support/ci/deploy.sh; fi
         - openssl aes-256-cbc -K $encrypted_50e90ce07941_key -iv $encrypted_50e90ce07941_iv -in ./support/ci/habitat-srv-admin.enc -out /tmp/habitat-srv-admin -d
-      script:
+      deploy:
         - sudo ./support/ci/deploy.sh
```

Comment:
From the travis docs:
>Note that pull request builds skip the deployment step altogether.
https://docs.travis-ci.com/user/deployment/#Pull-Requests